pkg/internal/checkup/executor/executor.go

```diff
@@ -139,16 +156,12 @@ func (e Executor) Execute(ctx context.Context, vmiName, podName, podContainerNam
 	log.Printf("traffic Generator port %d Packet output errors: %d", trafficDestPort, results.TrafficGeneratorInErrorPackets)
 
 	log.Printf("get testpmd stats in DPDK VMI...")
-	var testPmdStats map[string]int64
+	var testPmdStats []testPmdPortStats
```

Comment:
The usage of the keys to access a list is a bit odd, but if you prefer to use it like this, I think you should use an array and not a slice. This will assure that accessing the index will not blow up if the parser returns less items then expected.

Reply:
I find the use of keys is best here (it's the equivalent to enums..) 
I agree using an array and not a slice is a better practice. 
Changed, DONE.